client/curriculum/html/LinksAndButtons.tsx

```diff
@@ -0,0 +1,882 @@
+            Putem avea și URL-uri
+            {' '}
+            <strong>relative</strong>
+            , de obicei folosite
+            la realizarea meniurilor paginilor web:
+          </p>
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<a href="/team.html">Team</a>`}
+          />
+          <LessonFigure
+            withBorder
+            src="/images/lessons/links-and-buttons/menu-github.png"
+            alt="URL-uri relative"
+            demo="/demo/html/url-relativ"
+          />
+          <p>
+            Foarte utile sunt și link-urile care duc către o secțiune ale
+            aceleiași pagini sau așa numitele
+            {' '}
+            <strong className="formatted">jump links</strong>
+            .
+          </p>
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<a href="#Section-further-down"> 
+  Sari la secțiunea de mai jos 
+</a> 
+<h2 id="Section-further-down">Section further down</h2>
+            `}
+          />
+          <LessonFigure
+            withBorder
+            src="/images/lessons/links-and-buttons/jump-links.png"
+            alt="Link-uri care duc către o secțiune ale
+            aceleiași pagini sau așa numitele jump links"
+            demo="/demo/html/jump-links"
+          />
+          <LessonTip>
+            Observați atributul
+            {' '}
+            <strong className="formatted">ID</strong>
+            .
+            Aceasta este modalitatea prin care facem legătura între link și
+            secțiunea paginii la care vrem să ajungem.
+          </LessonTip>
+          <p>
+            Deși nu am ajuns la partea de CSS, merită menționat că jump links
+            trebuie să beneficieze de
+            {' '}
+            <a href="/concepts/smooth-scrolling">smooth scrolling</a>
+            {' '}
+            pentru ca
+            userii să aibă o experiență plăcută. Un usage case este chiar pe
+            platforma noastră:
+          </p>
+          <LessonFigure
+            isVideo
+            withBorder
+            src="/videos/lessons/links-and-buttons/smooth-scrolling.mp4"
+            alt="Redarea comportamentului de smooth scrolling"
+          />
+        </section>
+        <section>
+          <LessonHeading as="h4" id="alte-exemple-unde-vom-folosi-linkuri">
+            Alte exemple unde vom folosi link-uri
+          </LessonHeading>
+          <p>
+            Sunt situații în care vrem să oferim unui user posibilitatea de a
+            deschide email-ul pentru a trimite un mesaj nou. În cazul acesta ne
+            vom folosi de
+            {' '}
+            <strong className="formatted">mailto</strong>
+            :
+          </p>
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<a href="mailto:hello@frontend.ro">
+  Trimite email la Frontend.ro
+</a>`}
+          />
+          <LessonFigure
```

Comment:
Inline demo (like for the absolute URLs). Also show a video from mobile of how this works ( I think there are apps to record mobile screen. On iphone it can be done directly from the OS)